CWE/localisation/Elia's events.csv

```diff
@@ -99,3 +99,21 @@ EVT_8020442_DESC;As according to the regime's National Renewal policy, the first
+EVT_8020459_DESC;Having made no secret of his Monarchist sympathies, Generalissimo Amos Spiazzi has recently entered negotiations with the Savoy House, in particular with Victor Emmanuel, the son of late King Umberto II. Spiazzi and Victor have now been correspondants for a couple of years and the situation seems to have become favourable for the end of the exile imposed on the Royal House in 1948. The return of the Savoys will sign a final pacification with the monarchist side, who since the Referendum has been at odds with our government. Shall we go forward with the operations, the Savoys will take a state trip from their residence in Switzerland to Rome, where they will be greeted by the Generalissimo and taken to a tour of the Eternal City and later to a press conference.;;;;;;;;;;;;;x
+EVTDESC8020460;Having long suffered from heart problems, Generalissimo Amos Spiazzi has left us today at the age of 78. The country mourns the leader and a state funeral attended by NATO leadership will take place tomorrow in the morning in Spiazzi's native city, Trieste.;;;;;;;;;;;;;x
+EVTDESC8020461;At the late age of 87 years, Generalissimo Carlo Alberto dalla Chiesa has left us in his sleep earlier this night. Having made himself loved by the lower stratas of the population due to his efforts to fight organized crime and terrorism, dalla Chiesa's funeral is expected to be attended by thousands of people, many collaborators and even more survivors and families of people who have had their life threatened by the mafia. General dalla Chiesa surely will not be forgotten in the history text and his rule remembered as a stable period in Italian history.;;;;;;;;;;;;;x
+EVT_8020462_DESC;To counter the continuous expansion of organized crime in the South of the Country, Generalissimo dalla Chiesa, who has in the past already served in Sicily in a similiar role, has organized a mass operation, nicknamed Operation Sicilian Vespers, to end this problem once and for all. 120.000 troops will be deployed to the regions of Sicily and Calabria, respectively operation zones of Cosa Nostra and 'Ndrangheta, and the civil administration of the regions will be subordinated to a military council, hopeflly this will resolve our problems.;;;;;;;;;;;;;x
+
```

Comment:
There’s a blank line here. It needs to be removed or it might cause localisation errors.